installer/common/shared.sh

```diff
@@ -63,8 +63,9 @@ export zookeeperHosts=""
 
 # Spark Operator
 export sparkOperatorReleaseName="cloudflow-sparkoperator"
-export sparkOperatorChartVersion="0.4.0"
-export sparkOperatorImageVersion="1.3.0-M1-OpenJDK-2.4.4-0.8.2-cloudflow-2.12"
+export sparkOperatorChartVersion="0.6.2"
+export sparkOperatorImageName="lightbend/sparkoperator"
+export sparkOperatorImageVersion="1.3.0-M1-OpenJDK-2.4.4-1.0.1-cloudflow-2.12"
```

Comment:
@skonto Stavros, I think you built the current image tagged `1.3.0-M1-OpenJDK-2.4.4-0.8.2-cloudflow-2.12`. Could you also build a new image with tag `1.3.0-M1-OpenJDK-2.4.4-1.0.1-cloudflow-2.12` using the latest operator release? Or I can do it if you have a document that you can point me to for reference.

Reply:
@yuchaoran2011 sure here are the steps: https://github.com/lightbend/cloudflow/blob/master/external/spark/releaseSparkDist.sh#L65-L74, @debasishg since tags are related to a Cloudflow release should we somehow integrate this with the release scripts you have created (if not done yet)?

Reply:
Got it. I'll create and push the image and test it with this PR.

Reply:
I have created and pushed a new image with tag `1.3.0-M1-OpenJDK-2.4.4-1.0.1-cloudflow-2.12`. Now install (including dependent charts) and uninstall both work.